marshaller_test.go

```diff
@@ -485,6 +485,25 @@ var _ = Describe("Marshaller", func() {
 			})
 		})
 
+		Context("when all clauses are pointers to date time data types", func() {
```

Comment:
It will be good to add a testcase where timestamp pointer is nil and verify that it is not included in the where clause.

Reply:
Done.